spec/acceptance/autofs_init_spec.rb

```diff
@@ -1,6 +1,26 @@
 require 'spec_helper_acceptance'
 
 describe 'autofs' do
+  def cleanup_helper
```

Comment:
it could even be in a dedicated file if other test files needed it. and be included  with `require_relative '